django_react_proj/users-fe/src/components/Header.js

```diff
@@ -1,22 +1,20 @@
-import React, { Component } from "react";
+import React from "react";
 
-class Header extends Component {
-    render() {
+const Header = () => {
         return  (
             <div className="text-center">
             <img
               src="https://media.discordapp.net/attachments/821426521859162192/821456504875646976/teams-chaos.png?width=800&height=450"
               width="300"
               className="img-thumbnail"
-              style={{ marginTop: "20px" }}
+              style={{ marginTop: "40px" }}
```

Comment:
Try to keep styles in seperate file for each component (e.g.  Header.scss) and than import it in component file like:
`import styles from "./Header.scss"`. Then you can use it like: `style={styles.image}`.

Reply:
Done, I've created separete directories for each component.